data/s3dis/meta_data/anno_paths.txt

```diff
@@ -0,0 +1,272 @@
+Area_1/conferenceRoom_1/Annotations
```

Comment:
Is it a conventional way to use both txt files? Can we get the anno path from the name of data label? It seems they are closely related and have one-to-one correspondence.